block_my_feedback.php

```diff
@@ -132,7 +136,20 @@ public function fetch_feedback() : array {
 
         // Template data for mustache.
         $template = new stdClass();
+        $template->feedback = [];
+        $i = 0; // We only want to show up to 5 grades - so count the output.
+
         foreach ($submissions as $f) {
+            // Check if a quiz feedback should be shown.
+            if ($f->modname == 'quiz' && !$this->show_quiz_submission($f)) {
+                continue;
+            }
+
+            // Check if we have enough grades to show.
```

Comment:
Arbitrary, but this can go above the first if so we bail quicker when we have 5?